libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -368,6 +371,55 @@ public static boolean contentContainsGutenbergBlocks(String postContent) {
         return (postContent != null && postContent.contains(GUTENBERG_BLOCK_START));
     }
 
+    public static String replaceMediaFileWithUrl(Context context, @NonNull String postContent,
```

Comment:
As this is a static method, maybe it's a good candidate for some Util or Helper?